jekyll/_posts/2018-06-03-create-a-simple-macro.md

```diff
@@ -0,0 +1,120 @@
+
+---
+title: "Create a simple macro"
+author: Antonis Geralis
+---
+
+Hello, as you might know Nim is a powerful programming language that supports
+metaprogramming using macros. Though a lot of Nim programmers are unaware about
+the merits of them due to lack of learning resources. In the first part of
+this series we will discuss the use of macros to simplify the creation of
```

Comment:
I would recommend going through and removing any use of "we" in sentences. This sentence can be rewritten to:

"The first part of this series will discuss the ..."

It makes sentences more concise and thus easier to read.